app/controllers/hello_world_controller.rb

```diff
@@ -1,9 +1,13 @@
+# Set up the user properties. This user should appear on your LaunchDarkly users
+# dashboard soon after you run the demo. Normal and bootstrapped flag values for
+# this user should appear in your app.
+
 class HelloWorldController < ApplicationController
   def index
       user = {
-        key: "bob@example.com",
-        firstName: "Bob",
-        lastName: "Loblaw",
+        key: "example-user-key",
+        firstName: "Sandy",
+        lastName: "Sands",
```

Comment:
Do we have a standard for lastName?

Reply:
We've been removing `firstName` and `lastName` and replacing with `name: "Sandy"` (all three fields are available)